episodes/files/environment.yml

```
@@ -0,0 +1,11 @@
+name: dc-image
+channels:
+  - conda-forge
+dependencies:
+  - python>=3.11
+  - jupyterlab
+  - numpy
+  - matplotlib
+  - pandas
+  - scikit-image
+  - ipympl
```

```suggestion
  - ipympl
  - imageio
```
Should we add `imageio` here? It gets installed as a dependency of `scikit-image` anyway, but we do use it directly in quite a few episodes.

---

It won't hurt to include it in the list, I suppose 😄